network/datadog_checks/network/network.py

```diff
@@ -595,6 +589,46 @@ def _get_net_proc_base_location(proc_location):
             net_proc_base_location = proc_location
         return net_proc_base_location
 
+    def _read_int_file(self, file_location):
+        try:
+            with open(file_location, 'r') as f:
+                try:
+                    value = int(f.read().rstrip())
+                    return value
+                except ValueError:
+                    self.log.debug("Content of %s is not an integer", file_location)
+        except IOError as e:
+            self.log.debug("Unable to read %s, skipping %s.", file_location, e)
+            return None
+
+    def _get_iface_sys_metrics(self, custom_tags):
+        sys_net_location = '/sys/class/net'
+        sys_net_metrics = ['mtu', 'tx_queue_len']
+        try:
+            ifaces = os.listdir(sys_net_location)
+        except OSError as e:
+            self.log.debug("Unable to list %s, skipping %s.", sys_net_location, e)
```

Comment:
```suggestion
            self.log.debug("Unable to list %s, skipping system iface metrics: %s.", sys_net_location, e)
```
Nit- let's make the log line more descriptive